README.adoc

```diff
@@ -0,0 +1,53 @@
+= Eclipse uProtocol TCK Library 
+:toc:
+
+== Overview
+
+This library implements the uProtocol Test Compatibility Kit for the use of a Test Manager and Test Agents. A Test Manager will be responsible for sending commands to multiple connected Test Agents. The Test Agents, in turn, will carry out the tests specified in the commands, and return results back to the Test Manager for validation and processing.
+
+== Getting Started
+
+=== Prerequisites
+
+=== Importing the up-tck-python
+ 
+Set up upclient-socket-python local repository and install
+[source]
+----
+$ git clone https://github.com/eclipse-uprotocol/up-tck.git
+$ cd up-tck/python
+$ pip install .
+----
+*This will install the up-tck, making its classes and modules available for import in your python code.*
+
+=== Building Java Test Agent Jar
+
+If you would like to run any tests with the Java Test Agent, follow these instructions:
+
+1. Inside a terminal/command prompt, go to java_test_agent folder (cd java/java_test_agent).
+2. Run "mvn clean install". This will generate the JavaTestAgent JAR file under the target folder.
+
+=== Running BDD Tests
+
+For information about running BDD Tests, refer to https://github.com/hostilechild007/uprotocol-tck/blob/ulink_socket_python/BDD/README.adoc[BDD/README.adoc]
```

Comment:
Link is not correct. It should be https://github.com/eclipse-uprotocol/up-tck/BDD/README.adoc